src/_pytest/hookspec.py

```diff
@@ -609,3 +609,13 @@ def pytest_enter_pdb(config):
 
     :param _pytest.config.Config config: pytest config object
     """
+
+
+def pytest_leave_pdb(config):
```

Comment:
would it make sense to add the actual debugger as argument here?

Reply:
Good idea.
I'll add a new arg `pdb`.
The same might make sense for `pytest_enter_pdb` then maybe also?!

Reply:
Agree to both points. 👍